src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1475,6 +1495,16 @@ private function getErrorMessages(Exception $e)
         ];
     }
 
+    private function getPaymentErrorMessages(Exception $e)
+    {
+        return array_merge($this->getErrorMessages($e), [
+            InvalidArgumentException::class => $this->trans(
+                'Only numbers and decimal points (".") are allowed in the amount fields of the payment block, e.g. 10.50 or 1050.',
```

Comment:
Why is it (slightly) different from the first exception above, at line 1443? Can't we use the same string twice?

Reply:
@LouiseBonnard Because the exception line 1443 is not related to the payment block, it's a more global error message for other amount fields